### ISSUE TITLE
how to get gradient norm from keras_sequential_model

### ISSUE THREAD

**Issue by ds5j** [rstudio/keras#509]
Is there a way to get the norm of gradients computed by keras in a callback after each epoch in R?

**Comment by skeydan** [comment]
```callback_tensorboard``` has an option to write the gradients to disk:

```
callback_tensorboard(log_dir = NULL, histogram_freq = 0, batch_size = 32,
  write_graph = TRUE, write_grads = FALSE, write_images = FALSE,
  embeddings_freq = 0, embeddings_layer_names = NULL,
  embeddings_metadata = NULL, embeddings_data = NULL)
```
